metaseq/scripts/generation_benchmarks.py

```diff
@@ -0,0 +1,279 @@
+    ]
+    links_to_data["350m"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/350m/reshard.pt"
+    ]
+    links_to_data["1.3b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/1.3b/reshard-model_part-0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/1.3b/reshard-model_part-1.pt",
+    ]
+    links_to_data["2.7b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/2.7b/reshard-model_part-0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/2.7b/reshard-model_part-1.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/2.7b/reshard-model_part-2.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/2.7b/reshard-model_part-3.pt",
+    ]
+    links_to_data["6.7b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/6.7b/reshard-model_part-0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/6.7b/reshard-model_part-1.pt",
+    ]
+    links_to_data["13b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/13b/reshard-model_part-0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/13b/reshard-model_part-1.pt",
+    ]
+    links_to_data["30b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/30b/reshard-model_part-0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/30b/reshard-model_part-1.pt",
+    ]
+    links_to_data["66b"] = [
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-0-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-1-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-2-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-3-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-4-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-5-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-6-shard0.pt",
+        "https://dl.fbaipublicfiles.com/opt/v1_20220502/66b/reshard-model_part-7-shard0.pt",
+    ]
+
+    if key not in links_to_data.keys():
+        raise AssertionError(
+            "You passed model name "
+            + str(key)
+            + ". Please make sure that model name is in "
+            + str(list(links_to_data.keys()))
+        )
+
+    links_to_data["dependencies"] = [
+        "http://dl.fbaipublicfiles.com/metaseq_benchmark_dependencies.tar.gz"
+    ]
+
+    # download and unzip dependencies
+    for dependency in links_to_data["dependencies"]:
+        urllib.request.urlretrieve(dependency, "./dependencies.tar.gz")
+        file = tarfile.open("./dependencies.tar.gz")
+        file.extractall()
+
+    # download model checkpoint
+    for shard in links_to_data[key]:
+        file_name = shard.split("/")[-1]
+        urllib.request.urlretrieve(shard, "./dependencies/" + file_name)
+
+
+def download_prompts():
```

Comment:
instead of downloading prompts (and cleaning them up), I suggest we just write 10 (short) prompts and keep them as constants at the top of the file.

Reply:
Fair enough for 10 prompts but I was wondering if someone wanted more (the file has up to 1000) then it may just be too much clutter.  Also isn't it generally better to average out numbers from more prompts than from just 10?

Reply:
If all the prompts are the same length, I don't expect any variance at all. Really I think it's okay if we just have 1 prompt, and averaged over many runs of the same prompt.